src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
+$card-width: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 0) and (max-width: 487px) {
+    grid-template-columns: repeat(1, $card-width);
+  }
```

Comment:
you were told to use 1024px, 768px, 488px — so let's apply these numbers (neither less, nor greater)

Reply:
![image](https://user-images.githubusercontent.com/113321536/222653814-d69623b7-515c-456d-9e97-6ebffb64ff5d.png)
![image](https://user-images.githubusercontent.com/113321536/222653874-7ffadd03-d0a1-4f0c-a963-ade468a94fc3.png)
![image](https://user-images.githubusercontent.com/113321536/222654380-a8619bf8-122a-44c6-a3e2-63060f3a9b92.png)

but if I set exactly 1024, 768, and 488 it breaks task requirements. Checklist said that 4 columns should start at 1024px. Will it be okay, if my columns start at 489, 769, and 1025?